package-lock.json

```diff
@@ -31,6 +31,7 @@
         "jsonwebtoken": "^9.0.2",
         "jwks-rsa": "^3.2.0",
         "log4js": "^6.9.1",
+        "minio": "^8.0.5",
```

Comment:
We should not need minio client, we should use "plain" S3 client as we do not intend to use MinIO in deployments.